src/qt/editor/welcomewidget.cpp

```diff
@@ -157,33 +182,62 @@ using Role = WorkspaceTreeModel::Role;
 
 namespace {
 
+bool contains(std::string_view str, std::string_view substr) {
+    auto it = std::search(
+        str.begin(), str.end(),
+        std::boyer_moore_horspool_searcher(
```

Comment:
Interesting ;-)

Reply:
:) probably a premature optimization... but I might refactor it in the future